### PR TITLE
Unicode file path support on Windows

### PR DESCRIPTION
**Pull request by elasota**

### CODE REVIEW

**Comment by elasota** [comment]
Looks like gmock needs to be updated.

**Comment by sfiera** [comment]
I’ll handle that and disable the WINE build.

**Comment by sfiera** [comment]
I triggered a re-run after merging #11, but it looks like it’s still running the WINE build.

File: src/win/sfz/file.cpp
Comment (sfiera) [inline src/win/sfz/file.cpp:32]: Ah, I suppose I should’ve added the wide functions to the other classes in the string family. Can you add a TODO for me? (no need to block this PR on it)
```cpp
// TODO(sfiera): add cpp_wstr() et. al. to other string classes
// TODO(sfiera): s/copy().cpp_str()/cpp_str()/ throughout libsfz
```

**Review by sfiera — APPROVED** [review]

**Comment by sfiera** [comment]
Thanks!

**Pull request merged**
